.github/workflows/pull-request-review-comment.yml

```
@@ -0,0 +1,117 @@
+      github.event.comment.author_association == 'OWNER' &&
+      contains(github.event.comment.body, '@codex') &&
+      github.event_name == 'pull_request_review_comment'
+    permissions:
+      actions: read
+      contents: write
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Create GitHub App token
+        id: app-token
+        uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)